exercises/en/exc_03_09b.py

```diff
@@ -0,0 +1,7 @@
+import altair as alt
+import pandas as pd
+
+
+penguins = pd.read_csv('data/penguins.csv')
+
+___ = ___
```

Comment:
This is FAR too few hints. Maybe take this to where 9a is, after you add more hints to 9a.